pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -410,7 +410,34 @@ def default_compilation_pass_offline(
             passlist.append(FullPeepholeOptimise())
         elif optimisation_level == 3:
             passlist.append(RemoveBarriers())
-            passlist.append(AutoRebase({OpType.CX, OpType.H, OpType.Rz}))
+            passlist.append(
```

Comment:
Is this a relevant change, that should be in the changelog, too?

Reply:
I don't think we don't need to include this - all I've done is update the set of gates we `AutoRebase` to so that it exactly matches what `GreedyPauliSimp` accepts. We don't mention this rebase for the `GreedyPauliSimp` line of the changelog so I don't think it's worth explicitly pointing out.